docs/topics/proc_selecting-the-jdk-builder-image-for-your-application.adoc

```diff
@@ -0,0 +1,57 @@
+[id='selecting-the-jdk-builder-image-for-your-application_{context}']
+= Selecting a JDK image for your {runtime} application
+Edit your application `pom.xml` file to specify the image for building and running your {runtime} application.
```

Comment:
Edit your application `pom.xml` file--> Edit your application's `pom.xml` file?

Reply:
Edit the `pom.xml` file of/in your application. We're not supposed to use Saxon genitive with inanimate objects [IBM Style Guide].

Reply:
No Saxon's here.

Reply:
I was reacting to Sreelatha's comment. Still, I do believe _pom.xml file of/in your application_ is more correct here, or at least more consistent with the rest of the docs.

Reply:
@tradej. Understood and I revised based on your comment.